production/env.d/api.env

```diff
@@ -1,6 +1,6 @@
 # Warren API
 WARREN_API_SERVER_PORT=8100
-WARREN_LRS_HOSTS=http://ralph:8080
+WARREN_LRS_HOSTS=http://ralph-app-1:8080
```

Comment:
Is it really needed ? This is the name exposed by `docker` but not `docker compose` I guess